examples/basic/topology_optimization_cantilever.py

```diff
@@ -0,0 +1,179 @@
+""" .. _ref_topology_optimization:
+
+Topology Optimization of a Simple Cantilever
+---------------------------------------------------------------------------------------
+Description:
+
+This example demonstrates the Structural Optimization of simple
+cantilver. The Structural Analysis is performed with basic constraints and
+load then transferred to topology optimization.
```

Comment:
```suggestion
This example demonstrates the structural topology optimization of a simple
cantilever. The structural analysis is performed with basic constraints and
load, which is then transferred to topology optimization.
```

Reply:
Not sure if the last sentence should read as "The structural analysis is performed with basic constraints and is loaded and transferred to topology optimization."